packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -702,8 +702,7 @@ class GestureDetector extends StatelessWidget {
   Widget build(BuildContext context) {
     final Map<Type, GestureRecognizerFactory> gestures = <Type, GestureRecognizerFactory>{};
 
-    if (
-      onTapDown != null ||
+    if (onTapDown != null ||
       onTapUp != null ||
```

Comment:
nit: Indent the other lines so that they're aligned with the first line

Reply:
Thanks, fixed 🙂